app_specific_files/dummy_app_multicast/app_config.ini

```diff
@@ -0,0 +1,4 @@
+[DOCKER_PORT]
```

Comment:
Can we use the word example instead of dummy app?

Reply:
I generated this example using the automatic dummy app generator, so it is fixed  'dummy'. This is just used for testing. Should I remove this before merging?

Reply:
If we can make this a long-standing "example", we can keep it in there forever. There always needs to be one base example to showcase how things are done.

Reply:
We can do this.